docs/pmm/guides/sorts.md

```diff
@@ -0,0 +1,242 @@
+# Sort Options Guide
+
+There are various sort options available depending on what you want to sort.
+
+## ASCII Sort Order
+
+The first thing to note is that Plex Meta Manager utilizes the ASCII sort order, which defines which priority a character gets.
+
+![](images\sorts.jpg)
+
```

Comment:
I think this make it seem like this is determined by PMM, where really it's Plex enforcing it.